crew

```diff
@@ -14,7 +14,7 @@ require_relative 'lib/const'
 require_relative 'lib/util'
 
 # Add lib to LOAD_PATH
-$LOAD_PATH.unshift "#{CREW_LIB_PATH}lib"
+$LOAD_PATH.unshift "#{CREW_PREFIX}/lib/crew/lib"
```

Comment:
Why remove a constant already defined in `lib/const.rb`?

Reply:
`LOAD_PATH` don't works when I put `crew` in `#{CREW_PREFIX}/bin`, either the original and `"#{CREW_PREFIX}/lib/crew/lib"`. Solved when put it in `/usr/local/lib/crew/`.

Reply:
Changed back